src/components/articlesList.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import ArticleCard from "./articleCard";
+
+const ArticlesList = () => {
+  const [articles, setArticles] = useState([]); 
+  const [loading, setLoading] = useState(true); 
+
+  
+  useEffect(() => {
+    axios
+      .get("https://yahyas-northcoders-news-api.onrender.com/api/articles") 
+      .then((response) => {
+        console.log(response, "res")
+        setArticles(response.data.article); 
+        setLoading(false); 
+      })
+      .catch((error) => {
+        console.error("Error fetching articles:", error);
+        setLoading(false); 
+      });
+  }, []);
```

Comment:
This is good, but I would recommend having a look at how to create an instance in Axios:

Notes: l2c.northcoders.com/courses/sd-notes/front-end/#sectionId=axios,step=creating-an-axios-instance